components/header/index.tsx

```diff
@@ -0,0 +1,24 @@
+"use client"
+
+import { usePathname } from "next/navigation"
+import Link from "next/link"
+
+export const Header = () => {
+  const pathname = usePathname()
+  return (
+    <header className="bg-sky-400 text-white font-bold h-8 pl-5">
```

Comment:
It is better to set `py` instead of `height`. Also `pl-5`is not enough because you need `padding-right` too for small devices, for example.

So you will have smth like this `bg-sky-400 text-white font-bold p-6`.